dataactvalidator/validation_handlers/validator.py

```diff
@@ -72,6 +72,7 @@ def evaluateCrossFileRule(cls, rule, submissionId, record = None):
             fieldsToCheck = cls.cleanSplit(rule.rule_text_1,True)
             # For each entry, check for the presence of matching values in second table
             for thisRecord in sourceRecords:
+                print("thisRecord: " + str(type(thisRecord)) + ", " + str(thisRecord))
```

Comment:
@jworcestBAH Are the print statements in this file necessary? If not, they should be removed.